Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imdining",category=['you'],prompt="I'm Dining",pool=True,unlocked=True))
```

Comment:
I still think there could be a better word than dining. It seems a bit too formal to me. 

Maybe you could break up the topic into one for drinking and one for eating. That way, you can just say eating or drinking which sounds natural.